torch/distributed/nn/api/remote_module.py

```diff
@@ -69,16 +69,20 @@ def _instantiate_template(module_interface_cls, enable_moving_cpu_tensors_to_cud
     )
 
 
-def _create_module(module_cls, args, kwargs, device="cpu", module_interface_cls=None):
+def _create_module(module_cls, args, kwargs, device):
```

Comment:
Still keep the default value?

Reply:
this is an internal method, and the parameter is not used so removed altogether.

Reply:
As you said, it's fine that we don't add any default value here. However, the removal may trigger some test failures. May need to explicitly add some args to some tests.